e2e/wsl-integrations.e2e.spec.ts

```diff
@@ -164,129 +154,203 @@ test.describe('WSL Integrations', () => {
     }
   });
 
-  test.beforeAll(async() => {
-    createDefaultSettings();
-
-    electronApp = await startRancherDesktop(__filename, {
-      env: {
-        PATH:             path.join(workdir, 'system32') + path.delimiter + process.env.PATH,
-        RD_TEST_WSL_EXE:  path.join(workdir, 'system32', 'wsl.exe'),
-        RD_MOCK_WSL_DATA: path.join(workdir, 'config.json'),
-      },
+  function registerDistro(distro: string, files: Record<string, string> = {}, version = 2) {
+    test.beforeAll(async() => {
```

Comment:
So everytime you call `registerDistro` you create a new before-hook?

Reply:
Yep, but that made more sense when I had it _also_ call `afterAll` after each distro. At this point that makes less sense, and I should probably reorganize this a bit.